packaging/msi/SumoLogic.wixext/SumoLogicTests/ConfigUpdaterTests.cs

```diff
@@ -148,14 +183,40 @@ public void TestUpdate_Empty()
         public void TestUpdate_Invalid()
         {
             var filePath = Path.Combine(testDataPath, "invalid.yaml");
-            var config = new Config();
-            config.InstallationToken = "foobar";
+            var config = new Config { InstallationToken = "foobar" };
             config.SetCollectorFieldsFromTags(@"foo=bar,baz=kaz,xaz=yaz");
 
             using (MemoryStream ms = new MemoryStream())
             {
                 var configUpdater = new ConfigUpdater(new StreamReader(filePath));
             }
         }
+
+        [TestMethod]
+        public void TestUpdate_OpAmp()
+        {
+            var filePath = Path.Combine(testDataPath, "with-extensions-block.yaml");
+            var config = new Config { InstallationToken = "foobar", Ephemeral = true, RemotelyManaged = true, OpAmpFolder = "/opamp" };

```

Comment:
Just a few other tests that come to mind here
- RemotelyManaged set to `false` with the `OpAmpFolder` specified
- The `Ephemeral` option is set to `false`

Reply:
I guess the value of the `Ephemeral` option is not relevant to this test

Reply:
Added more tests were added for the following:
* Remotely Managed vs OpAmpFolder
* Ephemeral
* Api